CesiumIonClient/include/CesiumIonClient/Defaults.h

```diff
@@ -0,0 +1,95 @@
+#pragma once
+
+namespace CesiumIonClient {
+
+/**
+ * @brief The default assets.
+ */
+struct DefaultAssets {
+  /**
+   * @brief The asset id of the default imagery asset.
+   */
+  int64_t imagery = -1;
+
+  /**
+   * @brief The asset id of the default terrain asset.
+   */
+  int64_t terrain = -1;
+
+  /**
+   * @brief The asset id of the default buildings asset.
+   */
+  int64_t buildings = -1;
+};
+
+/**
+ * @brief A raster overlay available for use with a quick add asset.
+ */
+struct QuickAddRasterOverlay {
+  /**
+   * @brief The name of this asset.
+   */
+  std::string name{};
+
+  /**
+   * @brief The unique identifier for this asset.
+   */
+  int64_t assetId = -1;
+
+  /**
+   * @brief `true` if the user is subscribed to the asset, `false` otherwise.
+   */
+  bool subscribed = false;
+};
+
+/**
+ * @brief A quick add asset.
+ */
+struct QuickAddAsset {
+  /**
+   * @brief The name of this asset.
+   */
+  std::string name{};
+
+  /**
+   * @brief The name of the main asset. In most cases this will be the same as
+   * `name`, but in the cases of assets with raster overlays, this will be the
+   * non-imagery asset.
+   */
+  std::string objectName{};
+
+  /**
+   * @brief A Markdown compatible string describing this asset.
+   */
+  std::string description{};
+
+  /**
+   * @brief The unique identifier for this asset.
+   */
+  int64_t assetId = -1;
+
+  /**
+   * @brief This asset's type.
+   *
+   * Valid values are: `3DTILES`, `GLTF`, `IMAGERY`, `TERRAIN`, `KML`, `CZML`,
+   * `GEOJSON`.
+   */
+  std::string type{};
+
+  /**
+   * @brief `true` if the user is subscribed to the asset, `false` otherwise.
+   */
+  bool subscribed = false;
+
+  /**
+   * @brief The raster overlays available for this asset
+   */
+  std::vector<QuickAddRasterOverlay> rasterOverlays{};
+};
+
+struct Defaults {
+  DefaultAssets defaultAssets{};
+  std::vector<QuickAddAsset> quickAddAssets{};
+};
```

Comment:
Is `defaultAssets` or the `DefaultAssets` struct actually used for anything? Can we remove it and rename `Defaults -> DefaultAssets`? It would make this more intuitive to read in the Unreal Editor code

Reply:
I know this rename would be wide-reaching, so if you agree with the decision I am happy to do the name replacements on the Unreal / Unity side

Reply:
We don't currently plan to use the `DefaultAssets` for anything in Unreal or Unity, at least, but it's part of ion's `/v1/defaults` service, so we should include it. Also, as mentioned offline, I think we need to stick to the names that the service documentation uses, so we shouldn't rename `Defaults` to something else. It's namespaced (`CesiumIonClient::Defaults`) so hopefully not _too_ bad.